### ISSUE TITLE
Wrong DOI and URL in a publication entry

### ISSUE THREAD

**Issue by CodesByChris** [sg-dev/sg-website#3]
When I test-compile the website locally (`hugo server -D`), I get the following error. It already occurred in the version I pulled before I made my modifications today.
```
Error: Error building site: "[...]/sg-website/content/publications/2022/schweitzer2022the-role-of/index.md:1:1": URLs with protocol (http*) not supported: "https://www.worldscientific.com/doi/epdf/10.1142/S0219525922500047". In page "[...]/sg-website/content/publications/2022/schweitzer2022the-role-of/index.md"
```

Two questions:
1. Likely, the problem occurs because of the tag `url` in the linked `index.md`, which publication entries do not support. Shouldn't it be `url_pdf`?
2. When I follow the DOI `10.1142/S0219525922500047` mentioned in `index.md`, I'm linked to a different paper (Giulio Bottazzi and Daniele Giachini, "Strategically biased learning in market interactions"). Shouldn't the DOI be `10.1142/S0219525922500035`?

**Comment by CodesByChris** [comment]
When I fix these locally, `hugo server -D` still doesn't run through. I get this error:
```
Start building sites … 
hugo v0.101.0-d729475172621286ea48868f942de645bfcc16fa+extended linux/amd64 BuildDate=2022-08-13T17:30:23Z VendorInfo=snap
ERROR 2022/08/26 18:50:54 render of "page" failed: "[...]/themes/eth-hugo/layouts/team/single.html:46:7": execute of template failed: template: team/single.html:46:7: executing "main" at <partial "cards/list_card_publication.html" .>: error calling partial: "[...]/themes/eth-hugo/layouts/partials/cards/list_card_publication.html:7:25": execute of template failed: template: partials/cards/list_card_publication.html:7:25: executing "partials/cards/list_card_publication.html" at <dateFormat "2006" .Params.date>: error calling dateFormat: unable to cast <nil> of type <nil> to Time
ERROR 2022/08/26 18:50:54 render of "page" failed: "[...]/themes/eth-hugo/layouts/team/single.html:46:7": execute of template failed: template: team/single.html:46:7: executing "main" at <partial "cards/list_card_publication.html" .>: error calling partial: "[...]/themes/eth-hugo/layouts/partials/cards/list_card_publication.html:7:25": execute of template failed: template: partials/cards/list_card_publication.html:7:25: executing "partials/cards/list_card_publication.html" at <dateFormat "2006" .Params.date>: error calling dateFormat: unable to cast <nil> of type <nil> to Time
ERROR 2022/08/26 18:50:54 render of "term" failed: "[...]/themes/eth-hugo/layouts/projects/term.html:36:7": execute of template failed: template: projects/term.html:36:7: executing "main" at <partial "cards/list_card_publication.html" .>: error calling partial: "[...]/themes/eth-hugo/layouts/partials/cards/list_card_publication.html:7:25": execute of template failed: template: partials/cards/list_card_publication.html:7:25: executing "partials/cards/list_card_publication.html" at <dateFormat "2006" .Params.date>: error calling dateFormat: unable to cast <nil> of type <nil> to Time
ERROR 2022/08/26 18:50:54 render of "term" failed: "[...]/themes/eth-hugo/layouts/projects/term.html:36:7": execute of template failed: template: projects/term.html:36:7: executing "main" at <partial "cards/list_card_publication.html" .>: error calling partial: "[...]/themes/eth-hugo/layouts/partials/cards/list_card_publication.html:7:25": execute of template failed: template: partials/cards/list_card_publication.html:7:25: executing "partials/cards/list_card_publication.html" at <dateFormat "2006" .Params.date>: error calling dateFormat: unable to cast <nil> of type <nil> to Time
Error: Error building site: failed to render pages: render of "page" failed: "[...]/themes/eth-hugo/layouts/publications/single.html:9:50": execute of template failed: template: publications/single.html:9:50: executing "title" at <dateFormat "2006" .Params.date>: error calling dateFormat: unable to cast <nil> of type <nil> to Time
Built in 1080 ms
```


I think we can fix it by replacing
```
year: '2022'
```
with
```
date: '2022-05-25'
```
in `content/publications/2022/schweitzer2022the-role-of/index.md`.

After this, the website compiles perfectly fine for me.

**Comment by CodesByChris** [comment]
@verginer and I fixed the issue this morning in 606eed5169a5c93108bdd74e2bac04c96f3a4960. Thanks for the help!
